interfaces/python/CMakeLists.txt

```diff
@@ -267,13 +267,26 @@ install(
 # Standalone interface builds for wheels will need the files to be copied to a different location anyway,
 # so not copying the files for now isn't a big loss
 if(WIN32)
+    # Realistically for bundling into a Python wheel, the dll should be given a unique name
+    # to avoid conflicts with other packages using a different version of the dll with the same name
+    add_custom_command(
+        TARGET ${HELICS_PYTHON_TARGET_NAME} POST_BUILD
+        COMMAND
+            ${CMAKE_COMMAND} -E copy_if_different # which executes "cmake - E
+                                                  # copy_if_different..."
+            "$<TARGET_FILE:HELICS::helicsSharedLib>" # <--this is in- file
+            "$<TARGET_FILE_DIR:${HELICS_PYTHON_TARGET_NAME}>/" # <--this is out- file path
```

Comment:
So on Windows there is a distinction between the target linking file. (*.lib) and the target file (*.dll)  do you need both?  And if we are building afterwards on windows do we need the .lib file in a different location than it is already?

Reply:
Only the runtime needs to be in the wheel so the compiled Python module (a dll with a pyd extension) can find it. If someone is doing C/C++ development that needs the lib file for linking they should download the helics-shared archive for the release instead of using pip install.

The lib file is one of the reasons changing the library names gets annoying.

Reply:
The way building Python wheels works right now, I have GH actions download a copy of the compiled Helics-shared release archive and pull a copy of the HELICS source code into a subfolder. Then I have a setup.py script that uses CMake to do a standalone Python build for creating the precompiled packages, with the idea that we can also publish a source package that will let users with weird OS or Python version requirements build from source with pip install.